### ISSUE TITLE
Context menus show all items with ignoring context

### ISSUE THREAD

**Issue by kenz-gelsoft** [kenz-gelsoft/gecko-dev#78]
Right click shows unrelated menu items
- Add to dictionary
- Undo add to dictionary
- Copy Email Address

when no text selected and dictionary is not related

**Comment by kenz-gelsoft** [comment]
Where finally build up context menu items

https://searchfox.org/mozilla-esr128/source/browser/base/content/nsContextMenu.js

**Comment by kenz-gelsoft** [comment]
We want use browser toolbox again

https://github.com/kenz-gelsoft/gecko-dev/issues/60#issuecomment-2428144234

but this time may be harder as focusing the toolkit  closes context menu.

at least console will help.

**Comment by kenz-gelsoft** [comment]
We failed to execute WebAssembly as we disabled it

https://searchfox.org/mozilla-esr128/source/toolkit/components/translations/actors/TranslationsParent.sys.mjs#2888

**Comment by kenz-gelsoft** [comment]
esr128 latest release has fix for this.
```diff
~/src/gecko-dev> git diff toolkit/components/translations/actors/
diff --git a/toolkit/components/translations/actors/TranslationsParent.sys.mjs b/toolkit/components/translations/actors/TranslationsParent.sys.mjs
index acd033cc6ecf..0c3d4a9c5abf 100644
--- a/toolkit/components/translations/actors/TranslationsParent.sys.mjs
+++ b/toolkit/components/translations/actors/TranslationsParent.sys.mjs
@@ -2734,6 +2734,7 @@ export class TranslationsParent extends JSWindowActorParent {
  * Validate some simple Wasm that uses a SIMD operation.
  */
 function detectSimdSupport() {
+  try {
   return WebAssembly.validate(
     new Uint8Array(
       // ```
@@ -2757,6 +2758,9 @@ function detectSimdSupport() {
       ]
     )
   );
+  } catch {
+    return false;
+  }
 }
 
 /**
```

**Comment by kenz-gelsoft** [comment]
Fixed by updating to 128.5 based branch in https://github.com/kenz-gelsoft/gecko-dev/issues/79
